tslint.json

```diff
@@ -92,7 +92,7 @@
     "component-selector": [true, "element", "app", "kebab-case"],
     "use-input-property-decorator": true,
     "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "use-host-property-decorator": false,
```

Comment:
What's the purpose of it?

Reply:
Actually I am using `host` property to listen for click event on document and then I am directing it to `sugClose($event)` function to tell what is to be done. But then it results in linting error. It asks me to use Host Property Decorators i.e `@HostBindings` and `@HostListeners` to do the same task. Then I followed this answer https://stackoverflow.com/questions/45172565/use-hostbindings-instead-host-in-angular-4 and I came to know that it is fine to use host property. So I turned it to false. But now it is decreasing the code coverage.